ovs/lib/migration.py

```diff
@@ -245,5 +246,17 @@ def migrate():
                 vpool.metadata_store_bits = bits
                 vpool.save()
 
+        #######################################
+        # MDS safety granularity on vPool level
+        mds_safety_key = '/ovs/framework/storagedriver'
```

Comment:
Having this part out of band is risky. The mds checkup could be scheduled and expects these keys while the keys could not yet be set.